great-idea/css/index.css

```diff
@@ -153,4 +72,74 @@ h1, h2, h3, h4, h5 {
 
 h1 {
 	font-size: 80px;
-}
\ No newline at end of file
+}
+
+.container {
+	width: 880px;
+	margin: 0 auto;
+  }
+
+header {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+	width: 100%;
+}
+
+header nav {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+	margin-top: 35px;
+	width: 75%;
+  }
+
+header nav a {
+	margin-right: 65px;
+	font-size: 14px;
+	color: gray;
+	text-decoration: none;
+  }
+
+header nav a:hover {
+	text-decoration: underline;
+}
+
+
+.cta {
+	margin-top: 100px;
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+	justify-content: space-evenly;
```

Comment:
It looks like you're assigning the `justify-content` property and then overwriting it on the next line. This shouldn't cause any problems while rendering, but it might be confusing to look at later on.